packages/did-core-test-server/suites/implementations/did-algo.json

```diff
@@ -0,0 +1,86 @@
+{
+  "didMethod": "did:algo",
+  "implementation": "DID implementation build on top of the Algorand Blockchain and IPFS",
+  "implementer": "AID:Tech and Algorand Foundation",
+  "supportedContentTypes": [
+    "application/did+json",
+    "application/did+ld+json"
+  ],
+  "dids": ["did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef"],
+  "didParameters": {
+    "hl": "did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef?hl=zQmWvQxTqbG2Z9HPJgG57jjwR154cKhbtJenbyYTWkjgF3e",
+    "service": "did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef?service=algo-connect",
+    "relativeRef": "did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef?service=algo-connect&relativeRef=type",
+    "versionId": "did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef?versionId=0.1.0",
+    "versionTime": "did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef?versionTime=2021-10-20T03:19:33Z"
+  },
+  "did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef": {
+    "didDocumentDataModel": {
+      "properties": {
+        "@context": [
+          "https://www.w3.org/ns/did/v1",
+          "https://w3id.org/security/v1"
+        ],
+        "id": "did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef",
+        "created": "2021-10-20T03:19:33Z",
+        "updated": "2021-10-20T04:07:55Z",
+        "verificationMethod": [
+          {
+            "id": "did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef#master",
+            "type": "Ed25519VerificationKey2018",
+            "controller": "did:algo:f30304e2-e702-4f47-b242-2e9e064f6eef",
+            "publicKeyMultibase": "zCh9PDTZzeWxk2WdH4M1e8k2951D5D11jz7Uti9HRBGiK"
```

Comment:
Heads up, your public keys may be dropped by a json-ld document loader here because the contexts don't define these properties.

I'd suggest relying on the v3-unstable context to avoid this.

Reply:
err, I don't think v3-unstable is going to survive. I suggest using the pattern as shown in the latest did:key method: https://github.com/w3c/did-test-suite/blob/main/packages/did-core-test-server/suites/implementations/did-key-2020-db.json#L14-L18

Reply:
I couldn't find the proper suit that gave Ed25519VerificationKey2018 and publicKeyMultibase other than v3-unstable. However, I think your recommendation of upgrading to the 2020 suite is actually the better solution here.

Reply:
We have updated the contexts used to:

```json
"https://w3id.org/security/suites/ed25519-2020/v1",
"https://w3id.org/security/suites/x25519-2020/v1"
```